### PR TITLE
Keep a small fixed lidbloat so that documents does not move for small…

### PR DESCRIPTION
**Pull request by baldersheim**
… corpus sizes during tests.

@geirst PR

### CODE REVIEW

File: lib/app_generator/search_app.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ class SearchApp < App
                   :garbagecollectioninterval => :garbagecollectioninterval,
                   :doc_type => :doc_type,
                   :threads_per_search => :threads_per_search,
+                  :allowedlidbloat => :allowedlidbloat,
```
Comment (geirst) [inline lib/app_generator/search_app.rb:16]: Consider using snake case (as elsewhere): allowed_lid_bloat

**Review by geirst — APPROVED** [review]
👍

File: lib/app_generator/search_app.rb (hunk 1 above)
Comment (baldersheim) [reply lib/app_generator/search_app.rb]: Snake it is.

**Review by geirst — COMMENTED** [review]
👍

**Pull request merged**